docs/source/references.bib

```diff
@@ -315,6 +298,36 @@ @Article {liu-2018-first
   eprint =	 {1709.00106}
 }
 
+@InProceedings {dabov-2008-image,
```

Comment:
Entries were ordered in alphabetical order of cite keys.

Reply:
Sorry about that, I didn't realized that. I tried to put the bm3d+bm4d entries together.

Reply:
Not a big deal, and it's already fixed.

Reply:
Thank you for fixing it.